### PR TITLE
Fix wasBlockJustInserted when adding grid items

### LINKED ISSUE #97: wasBlockJustInserted fails when adding grid items
### Description
In #75 we're using `wasBlockJustInserted` to make sure newly added blocks don't run the migration and get the old defaults as static values.

This fails when using the Grid block. When you choose a selected layout, only the last block inserted returns true for `wasBlockJustInserted`, making it so the previous grid items have those old defaults saved as static values.

### Steps to reproduce
1. Using #75, add a Grid block
2. Choose a layout with more than one column

#### Expected behavior
All grid items should look the same.

#### Actual behavior
All grid items except for the last one have our old 40px of padding.

### PR DESCRIPTION
**Pull request by tomusborne**
This fixes #97.

It adds a `wasBlockJustInserted` attribute to the Container block set to `false` by default.

When adding grid items using the Grid block, it sets this attribute to `true`.

Once the Container block has mounted, we set it back to its default so the attribute doesn't remain in the comment delimiter.

### CODE REVIEW

File: src/blocks/container/edit.js [hunk 1]
```diff
@@ -133,7 +132,7 @@ class GenerateBlockContainer extends Component {
 
 		// Set our old defaults as static values.
 		// @since 1.4.0.
-		if ( ! wasJustInserted && isBlockVersionLessThan( this.props.attributes.blockVersion, 2 ) ) {
+		if ( ! wasBlockJustInserted && isBlockVersionLessThan( this.props.attributes.blockVersion, 2 ) ) {
```
Comment (JeanPaiva) [inline src/blocks/container/edit.js:135]: This should not be called as `wasBlockJustInserted( props )` to get the correct value? Cause you imported the function, and are not using it from the props the way you were doing before.

File: src/blocks/container/edit.js [hunk 2]
```diff
@@ -163,6 +162,10 @@ class GenerateBlockContainer extends Component {
 			}
 		}
 
+		if ( wasBlockJustInserted ) {
```
Comment (JeanPaiva) [inline src/blocks/container/edit.js:165]: Same as above.

**Review by JeanPaiva — COMMENTED** [review]
The tests are working as expected.

File: src/blocks/container/edit.js (hunk 1 above)
Comment (tomusborne) [reply src/blocks/container/edit.js:135]: Nice catch.. Now when I do this, the `wasBlockJustInserted` prop is `false` even when freshly inserting it.. Wonder why it stopped working all of a sudden.

Comment (JeanPaiva) [reply src/blocks/container/edit.js:135]: But this check is actually being always true, cause it checks if `wasBlockJustInserted` is defined. I'll check this behavior to see if I can catch something

Comment (tomusborne) [reply src/blocks/container/edit.js:136]: Yea, that's definitely a bug.

What's weird is that even when fixing that, `select( 'core/block-editor' ).wasBlockJustInserted( props.clientId )` is now `false` even when inserting it as a fresh block. Not sure why that stopped working..

Comment (tomusborne) [reply src/blocks/container/edit.js]: Hmm, seems to be working now... Let me know how it works for you.

**Review by JeanPaiva — APPROVED** [review]
It's working as expected.

**Pull request merged**